package.json

```diff
@@ -3,100 +3,54 @@
   "version": "1.0.0",
   "main": "server/server.js",
   "engines": {
-    "node": "6.11.1",
-    "npm": "3.10.x"
+    "node": "8.9.4",
+    "npm": "5.6.x"
   },
   "scripts": {
-    "clean": "rimraf client/dist/",
-    "heroku": "npm run clean && cross-env NODE_ENV=production npm start",
-    "start": "node .",
-    "lint": "eslint .",
-    "pretest": "npm run lint",
-    "posttest": "nsp check --threshold 4",
-    "test": "jest",
-    "storybook": "start-storybook -p 6006 -c client/stories -s client",
-    "build-storybook": "build-storybook"
+    "start-js": "react-scripts start",
+    "start-server": "node ./server/server.js",
+    "start": "npm-run-all -p watch-css start-js start-server",
+    "build-js": "react-scripts build",
+    "build": "npm-run-all build-css build-js && cpy build/index.html build --rename 404.html",
+    "test": "react-scripts test --env=jsdom",
+    "eject": "react-scripts eject",
+    "build-css": "node-sass-chokidar src/ -o src/",
+    "storybook": "start-storybook -p 9001 -c .storybook",
```

Comment:
Let's add a `postinstall` script to automatically install the server dependencies after `npm install`

Reply:
The script can be `npm install --prefix server`